src/Cake.Common/Tools/XUnit/XUnit2SettingsExtensions.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Cake.Common.Tools.XUnit
+{
+    /// <summary>
+    /// Contains functionality related to XUnit2 settings.
+    /// </summary>
+    public static class XUnit2SettingsExtensions
+    {
+        /// <summary>
+        /// Adds a trait to the settings, to include in test execution.
+        /// </summary>
+        /// <param name="settings">The settings.</param>
+        /// <param name="name">The trait name.</param>
+        /// <param name="values">The trait values.</param>
+        /// <returns>The same <see cref="XUnit2Settings"/> instance so that multiple calls can be chained.</returns>
+        public static XUnit2Settings IncludingTrait(this XUnit2Settings settings, string name, params string[] values)
```

Comment:
The name (`IncludingTrait`) is good, but we might want to follow other Cake aliases and use `WithTrait` here.
What do you think?